### PR TITLE
bf: S3C-1286 increase bucketname length

### PR DESCRIPTION
**Pull request by dora-korpar**
Increases bucketname length to 255 characters for 7.4 and above

### CODE REVIEW

**Comment by dora-korpar** [comment]
@bert-e reset

**Review by miniscruff — APPROVED** [review]

File: lib/s3routes/routesUtils.js [hunk 1]
```diff
@@ -886,7 +886,7 @@ const routesUtils = {
         // eslint-disable-next-line no-useless-escape
         const dnsRegex = new RegExp(/^[a-z0-9]+((\.|\-+)[a-z0-9]+)*$/);
         // Must be at least 3 and no more than 63 characters long.
```
Comment (jonathan-gramain) [inline lib/s3routes/routesUtils.js:888]: The comment does not reflect the changed value anymore

**Review by rachedbenmustapha — COMMENTED** [review]
Copy-paste from my comment in JIRA:

What’s the rationale for becoming incompatible with AWS S3 and incompatible with standard DNS?

According to [their docs](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html) the 255 size limit was a 1-region glitch and was since deprecated, coincidentally 4 days before this ticket was opened:

> “Before March 1, 2018, buckets created in the US East (N. Virginia) Region could have names that were up to 255 characters long and included uppercase letters and underscores. Beginning March 1, 2018, new buckets in US East (N. Virginia) must conform to the same rules applied in all other Regions.”

**Comment by rahulreddy** [comment]
Good catch @RachedBenMustapha !

**Comment by damienverine-scality** [comment]
Can you close this PR as we closed the Jira ticket as "Won't Fix" ?

**Pull request closed without merging**